spring-integration-smb/src/main/java/org/springframework/integration/smb/session/SmbSession.java

```diff
@@ -498,32 +494,6 @@ public SmbFile createSmbDirectoryObject(String _path) throws IOException {
 		return createSmbFileObject(_path, true);
 	}
 
-	/**
-	 * Static configuration of the JCIFS library.
-	 * The log level of this class is mapped to a suitable <code>jcifs.util.loglevel</code>
-	 */
-	static void configureJcifs() {
```

Comment:
So, don't we need to reconfigure logging for `jcifs` any more?
What drove you to drop this off?

Reply:
Well they are using SLFJ now and not their embedded, homegrown logger that wasn't really compliant with Log4J and Commons Log. We can configure the logging level easily using config files now in JCIFS.

Reply:
Good. Would be great to reflect that in the commit message.
Thanks

Reply:
Sorry should have stated that... Thanks!